app.py

```diff
@@ -1,41 +1,137 @@
-from flask import Flask, request, render_template, redirect
+import os
+from flask import Flask, request, render_template, redirect, session
+from flask_bcrypt import Bcrypt
+from dotenv import load_dotenv
+from models import db, User
+<<<<<<< Updated upstream
```

Comment:
Upon more investigation, I see this issue is likely due to a fork being created by @harshicabalu at [this link](https://github.com/hbalu/itsc-3155-FromScratch/tree/session-implementation-2). Since we are the main contributors to this repository, we do not need to fork the repository at all; we can make branches and commit directly to those branches in this repository. Please refrain from forking this repository in the future, as it can cause issues like merge conflicts.